src/app/interface/dataset.interface.ts

```diff
@@ -26,3 +26,8 @@ export interface DatasetsAccountResponse {
     pageInfo: PageBasedInfo;
     datasetTotalCount: number;
 }
+
+export interface DatasetRequestBySql {
+    query: string;
```

Comment:
Most imporant: you are not using the "skip" argument that was designed specially for this ticket.